src/app/static/src/js/components/runReport/reportList.vue

```diff
@@ -61,6 +62,12 @@
                 return this.reports.sort((a, b) => a.name.localeCompare(b.name));
             }
         },
+        mounted () {
+            if(this.initialSelectedReport) {
+                this.query = this.initialSelectedReport
+                this.$emit("update:report", this.initialSelectedReport)
```

Comment:
Without emitting an update, nothing gets selected.

Reply:
Yes but it might not be valid to emit if the report is not in the branch. You'll need to check if it's actually in the list of reports provided in the props.